src/apb/engine.py

```diff
@@ -29,9 +32,44 @@ def load_dockerfile(df_path):
         return dockerfile.readlines()
 
 
-def load_example_specfile():
-    with open(EX_SPEC_FILE_PATH, 'r') as spec_file:
-        return spec_file.readlines()
+def convert_params_to_dict(params):
+    param_dict = {}
+    param_list = []
+
+    for param in params:
```

Comment:
@jmrodri agreed it's a list, but on line 40 and 43 we're splitting strings by ',' and '='.  It just seems strange because then the list seems to be a bunch of strings that look like I mentioned above. I'm just wondering if we could gather the data in a better way so we don't have to loop through a list of strings that already seem to be in key value pairs.